pipelines/optimus/README.md

```diff
@@ -109,50 +118,63 @@ The tools each Optimus task employs are detailed in the following table:
+UMIs are designed to distinguish unique transcripts present in the cell at lysis from those arising from PCR amplification of these same transcripts. But, like cell barcodes, UMIs can also be incorrectly sequenced or amplified. The [UmiCorrection](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/UmiCorrection.wdl) task uses [Umi-tools v.0.0.1](https://pypi.org/project/umi-tools/0.0.1/) to apply a network-based, "directional" correction method ([Smith, et al., 2017](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5340976/)) to account for such errors. This task makes UMI corrections to alignments made with the 'GE' tag. This step will add a 'UB' tag for UMI-corrected barcodes. 
 
 ### 6. Summary Metric Calculation
 
 The [Metrics](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/SequenceDataWithMoleculeTagMetrics.wdl) task uses [sctools](https://github.com/HumanCellAtlas/sctools) to calculate summary metrics which help assess the quality of the data output each time this pipeline is run. These metrics are included in the Zarr and Loom output files. A detailed list of these metrics is found in the [Loom_schema documentation](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/Loom_schema.md).
 
 ### 7. Expression Matrix Construction
 
-The Optimus [Count](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/CreateCountMatrix.wdl) task evaluates every read in the BAM file and creates a UMI-aware expression matrix using [Drop-seq tools](https://github.com/broadinstitute/Drop-seq). This matrix contains the number of molecules that were observed for each cell barcode and for each gene. The task discards any read that maps to more than one gene, and counts any remaining reads provided the triplet of cell barcode, molecule barcode, and gene name is unique, indicating the read originates from a single transcript present at the time of cell lysis.
+The Optimus [CreateCountMatrix](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/CreateCountMatrix.wdl) task (imported as "Count") evaluates every read in the BAM file and creates a UMI-aware expression matrix using [Drop-seq tools](https://github.com/broadinstitute/Drop-seq) and [sctools](https://github.com/HumanCellAtlas/sctools). This matrix contains the number of molecules that were observed for each cell barcode and for each gene. The task discards any read that maps to more than one gene, and counts any remaining reads provided the triplet of cell barcode, molecule barcode, and gene name is unique, indicating the read originates from a single transcript present at the time of cell lysis. To correctly specific the gene name tag, this task will look for the 'GE' tag (counting_mode = 'sc_rna') or 'gn' tag (counting_mode = 'sn_rna').
```

Comment:
The last sentence here seems a little off. Maybe it should be "To correctly specify..."?

Reply:
Comment still stands

Reply:
Thanks for the catch! I sentence was out of date (no longer using the 'gn' tag, so I removed it.